crud/users.py

```diff
@@ -21,13 +21,69 @@ async def create_user(user: users_schema.UserCreate):
     if db_user:
         raise HTTPException(status_code=400, detail="Email already registered")
 
+    query = users_model.user.select()
+    stored_data = await database.fetch_one(query)
+    if stored_data == None: is_admin = True
```

Comment:
So the admin is the first user created only?

Reply:
Yes, the first created user is admin to gain right add/change another users. After that he can add user and additionally give him admin rights.